src/github.com/matrix-org/go-neb/services/github/github.go

```diff
@@ -24,30 +24,27 @@ var ownerRepoIssueRegex = regexp.MustCompile("([A-z0-9-_]+)/([A-z0-9-_]+)#([0-9]
 
 type githubService struct {
 	id                 string
+	serviceUserID      string
 	webhookEndpointURL string
-	BotUserID          string
 	ClientUserID       string
 	RealmID            string
 	SecretToken        string
+	HandleCommands     bool
+	HandleExpansions   bool
```

Comment:
Should this be part of the Service interface so that they can be indexed in the database?